test/ventilation_test.h

```diff
@@ -6,3 +6,11 @@
 int startVentilation_afterOneInspiration_openInspiratoryValve();
 
 int startVentilation_duringInspiration_doesNotopenInspiratoryValve();
+
+int startVentilation_afterOneInspiration_setsVentilationPhaseToInspiration();
```

Comment:
the function under test is called `start_inspiration`... I am not sure why these tests are named this way. Please rename them all to fit the convention.

Reply:
done